app/misc/repo.ts

```diff
@@ -274,41 +276,55 @@ function openRepository() {
         //Get the list of branches from the repo
         return repository.getReferences(Git.Reference.TYPE.LISTALL);
       })
-      .then(function (branchList) {
```

Comment:
Many of the changes in this section are just changing names from `branch.*` to `ref.*` to reflect the fact that these variables are in fact git refs and not specifically branches